charts/app-config/values-production.yaml

```diff
@@ -2557,6 +2557,30 @@ govukApplications:
 
   - name: smokey
     chartPath: charts/smokey
+    helmValues:
+      cronJobs:
+        - name: smokey
+          schedule: "*/10 * * * *"
+          args:
+            - cucumber
+            - --profile={{ .Values.govukEnvironment }}
+            - --strict-undefined
+            - --tags=not @not{{ .Values.govukEnvironment }}
```

Comment:
The [profiles](https://github.com/alphagov/smokey/blob/aadea2b/config/cucumber.yml) already do this. (Similarly in values.yaml below.)
```suggestion
```

Reply:
Ah yes, good shout. I've also removed the `profile` from the `smokey-failover` job, where there is currently no corresponding profile (so are setting the tags explicitly).